### DISCUSSION TITLE
Controlling OpenTelemetry Span Transmission for Specific Logic Flows in Spring Boot

### DISCUSSION THREAD

**Discussion by GalaxyiPhone** [open-telemetry/opentelemetry-java-instrumentation#12790] [Q&A]
Hello,
I'm currently using the OpenTelemetry Spring Boot Starter in my project.

I have a question.
Traces and spans are automatically being sent to the collector through OpenTelemetry JDBC. However, I'd prefer if traces and spans were not sent from JDBC for certain logic flows.

For example, in a logic flow of Foo -> Bar -> Baz, if spans are sent during the Bar/Baz stages, I'd like to have the option where spans are not sent during the Bar/Baz stages for a different flow like Fie -> Bar -> Baz, even though the Bar/Baz logic is the same.

To summarize my question: 
I'm wondering if there are any options or configurations available to disable span transmission for the latter part of the logic while keeping the logic itself unchanged, based on the initial part of the flow.

**Comment by KimDoubleB** [comment]
I think this can be solved through `SamplerCustomizer`.
You can disable sampling via `SamplingResult.drop` in special case cases where you don't want to send traces.

ref
- https://opentelemetry.io/docs/zero-code/java/spring-boot-starter/sdk-configuration/#programmatic-configuration
